src/utils/tezos/helpers.test.ts

```diff
@@ -263,29 +264,45 @@ describe("tezos utils helpers", () => {
   });
 
   describe("operationsToBatchParams", () => {
+    const executeParams = {
```

Comment:
you repeated those so many times, maybe it's worth having a fixture for it defined? :)